chia/wallet/coin_selection.py

```diff
@@ -0,0 +1,53 @@
+import random
+from typing import Set, Optional, Tuple, List
+from chia.types.blockchain_format.coin import Coin
+from chia.util.ints import uint64
+from chia.wallet.wallet_coin_record import WalletCoinRecord
+
+
+# we use this to check if one of the coins exactly matches the target.
+def check_for_exact_match(coin_records: List[WalletCoinRecord], target: int) -> Optional[WalletCoinRecord]:
+    for coinrecord in coin_records:

```

Comment:
use `camel_case` naming so `coin_record`, the same everywhere. For classes we can use ThisKindOfName